homeassistant/components/zha/config_flow.py

```diff
@@ -151,7 +146,7 @@ def hass(self):
         return self._hass
 
     @hass.setter
-    def hass(self, hass):
+    def hass(self, hass: HomeAssistant):
```

Comment:
If we type methods, we require them to become fully typed. This method is missing a return value.

Reply:
done